src/components/Causes/CausesDropInPaymentWrapper.vue

```diff
@@ -110,10 +112,48 @@ export default {
 				});
 		},
 		doBraintreeCausesSubscription(nonce, deviceData, paymentType) {
-			/** ! TODO add if (this.action === 'Update') logic here similar to
-			* src/components/MonthlyGood/MonthlyGoodDropInPaymentWrapper.vue
-			* for editing a cause subscription GD-155
-			*/
+			if (this.action === 'Update') {
+				console.log(nonce, deviceData, paymentType);
```

Comment:
bump

Reply:
🤦  Did we not have a lint warning for this at one time?

Reply:
I thought we had a warning but it def doesn't block anything...I don't see it either at this point in the builds